governance/roles.md

```diff
@@ -145,8 +148,32 @@ and led by a Security Assessment Facilitator, who will:
   of significant process changes).
 * triage issues related to security assessments.
 
+### Meeting facilitator
 
-### Triage Team
+The group meetings are an important part of community building and the 
+facilitator ensures a welcoming and inclusive atmosphere. In keeping with these
+goals, the meeting facilitator has the following responsibilities:
+* prepares the meeting notes with template and agenda
+* coordinates with presenter, if applicable
+* ensures there are scribes
+* runs meeting check-in, including partner groups
+* leads the meeting through the agenda
+
+Prerequisites:
+* active member
+* history of regular attendance
+* prior participation as scribe
+
+One or more of the following:
+* served in another facilitator role
+* leader on a project team
+* tech lead
+* co-chair
+
+Any member who meets the above qualifications is encouraged to add themselves
+to the [list of people in this role](.github/settings.yml).
```

Comment:
This link is actually going into relative `governance/` probably `/.github/settings.yml` would work?